internal/storetest/remotetest.go

```diff
@@ -34,19 +34,26 @@ func RunRemoteCheckTest(
 ) []ModelTestCheckSingleResult {
 	results := []ModelTestCheckSingleResult{}
 
-	for relation, expectation := range checkTest.Assertions {
-		result := RunSingleRemoteCheckTest(
-			fgaClient,
-			client.ClientCheckRequest{
-				User:             checkTest.User,
-				Relation:         relation,
-				Object:           checkTest.Object,
-				Context:          checkTest.Context,
-				ContextualTuples: tuples,
-			},
-			expectation,
-		)
-		results = append(results, result)
+	users := GetEffectiveUsers(checkTest)
+	objects := GetEffectiveObjects(checkTest)
+
+	for _, user := range users {
```

Comment:
Minor - this code is repeated - we probably want to abstract it in a later follow up